index.js

```diff
@@ -59,8 +77,15 @@ let defaults = require('./defaults.js');
 // `null` in the primary process. In the child process it resolves as
 // documented above.
 
+// The actual entry point, a wrapper that enables the telemetry and starts the
+// root span
 module.exports = async function(options) {
+  const telemetry = opentelemetry(options);
+  let spanName = 'apostrophe:boot';
```

Comment:
should be `const` I think.

Reply:
It is a mutable variable, see the `options.cluster`. eslint is normally auto fixing such things for me :)